app/javascript/styles/_index.scss

```diff
@@ -515,7 +509,16 @@ html {
     }
   }
 
-  .ui-widget-overlay {
+  // Fix z-index issues because calculations for Branch FlowConnectorPath elements start
+  // their lines from centre of a Branch Node, which places them directly over those
+  // elements, without this workaround in place (easier to fix z-index than calculations).
+  // Also includes some elements that gained position:fixed because they fell off the view.
+  .FlowConnectorLine,
+  .FlowConnectorPath,
+  .govuk-header,
+  .govuk-navigation-app,
+  .ui-widget-overlay,
+  .fb-preview-button {
     z-index: 1;
```

Comment:
This bit may potentially cause some issues - but I could be wrong.  Lets get it merged and go from there!

Reply:
Agreed. I think this is a horrible solution and would be happy for alternatives.